donation-api/src/main/java/org/interfaithsanctuary/donationapi/model/Donor.java

```diff
@@ -41,6 +41,10 @@
     @Column(name = "donor_join_date")
     private Date joinDate;
 
+    @OneToMany(mappedBy = "donor", cascade = CascadeType.ALL)
+    @JsonIgnore
+    private Set<Donation> donations;
+
     public Long getId() {

```

Comment:
Does the `id` field come from somewhere else? I don't see it in the model

Reply:
The section of code I added here was to represent that one donor can give many donations. The mapped by shows that it is related to donation by the "donor" field in donations.

Reply:
Hopefully that answered the question but if not let me know :)

Reply:
Oops, for some reason the `id` wasn't showing up in the review code so I thought it got deleted (or I didn't see it). Checked the Donor model in your branch and saw it though so we're good 👍